packages/langium/src/documentation/index.ts

```diff
@@ -0,0 +1,8 @@
+/******************************************************************************
+ * Copyright 2023 TypeFox GmbH
+ * This program and the accompanying materials are made available under the
+ * terms of the MIT License, which is available in the project root.
+ ******************************************************************************/
+
+export * from './documentation-provider';
+export * from './jsdoc';
```

Comment:
We don't have such index files for all subfolders. Do we want that in general?

Reply:
I believe it might make sense, yeah.